tests/test_datacard.py

```diff
@@ -1,9 +1,42 @@
+import os
+import posixpath
 import re
 
+import matplotlib.pyplot as plt
+import numpy as np
 import pytest
 
+import audeer
+import audiofile
+import audplot
+
 import audbcards
 from audbcards.core.dataset import create_datasets_page
+from audbcards.core.utils import set_plot_margins
+
+
+BUILD = audeer.path('..', 'build', 'html')
+
+
+def test_datacard_example(db, cache):
+
```

Comment:
There is no one-line docstring yet.

Reply:
The test is also becoming quite long I find.  One could  document the three assertions made, or refactor into separate tests, or pus some of the creation logic into fixtures, or a mixture of all of these mechanisms.

Reply:
I guess you are referring to the fact that we run

```python
dataset = audbcards.Dataset(pytest.NAME, pytest.VERSION, cache)
datacard = audbcards.Datacard(dataset)
```
inside `test_datacard_example()`, `test_datacard_lines_similar()` and `test_datacard_player()`.
This should indeed be changed.
I think this is related to https://github.com/audeering/audbcards/issues/35 and would require too much work to incorporate it in this pull request.

---

I added a docstring.